tests/expected/function-contract/history/copy_pass.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
```

Comment:
It would be nice if test cases had a little description about what is intended behavior to be tested here. Sometimes the test description contains an error message that is very obvious, which might ultimately making this extra description unnecessary. So, you can use your judgment where to add it.